tailwind.config.js

```diff
@@ -1,75 +1,77 @@
-import animatePlugin from 'tailwindcss-animate';
-
 /** @type {import('tailwindcss').Config} */
-
-export default {
-  darkMode: ['class'],
-  content: ['./index.html', './src/**/*.{ts,tsx,jsx,js}'],
-  prefix: '',
+module.exports = {
+  darkMode: ["class"],
+  content: [
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
+  ],
+  prefix: "",
   theme: {
     container: {
       center: true,
-      padding: '2rem',
+      padding: "2rem",
```

Comment:
is `' '` caused some issues or what? Why have we changed to `" "` ?

Reply:
autogenerated by shadcdn. Can revert this if needed

Reply:
Yes. Please. And we could think to maybe include configs into prettier. And work would be done by prettier 🤔 
We will see. Comment marked as Resolved